hello_world/blockchain_app/genesis-block.json

```diff
@@ -0,0 +1,3395 @@
+{
```

Comment:
Please check if its still required with latest release of sdk.

Reply:
@nazarhussain I checked to run the application with the default devnet genesis block, but when I try to send a transaction it fails with error
`10:18:42 ERROR lisk-framework: Failed to add transaction to pool. (module=lisk:app)
Message: Failed to validate signature '8d8668503f6a5398c8f637a9ca7db53160450160af92b26e6b9d728415bc5ac49fcc2c3d454da3693e5a6cdff070fde754dac4711d157e8f74ec100c9aac9f0b' for transaction with id 'a2cc88d5ce6bf21409035c923a2a0d3499596004c8564fd635c3b4c444104c21' 
 Trace: TransactionApplyError: Failed to validate signature '8d8668503f6a5398c8f637a9ca7db53160450160af92b26e6b9d728415bc5ac49fcc2c3d454da3693e5a6cdff070fde754dac4711d157e8f74ec100c9aac9f0b' for transaction with id 'a2cc88d5ce6bf21409035c923a2a0d3499596004c8564fd635c3b4c444104c21'
    at Processor.verifyTransactions (/Users/mona/git/lisk-sdk-examples/hello_world/blockchain_app/node_modules/lisk-framework/dist-node/node/processor/processor.js:222:23) `